app/js/components/connection_request_widget.ts

```diff
@@ -107,6 +109,17 @@ class CollectionWidget {
     });
   }
 
+  private deleteFields() {
+    const $inputContainer = $(this.prototype.replace(/__name__/g, this.index.toString()));
```

Comment:
Is it a requirement to do the `replace` action here? Seems to me you just want to clear their values and be done with it?